modules/tinymce/src/plugins/lists/main/ts/actions/ToggleList.ts

```diff
@@ -59,6 +61,53 @@ const getEndPointNode = (editor: Editor, rng: Range, start: Boolean, root: Node)
     container = container.nextSibling;
   }
 
+  // The reason why the next two if statements exist is because when the root node is a table cell (possibly some other node types)
```

Comment:
Thanks for this comment. I'm starting to see what you're doing now, which helps with the review process.